{"uuid":"bffb0f69-a9cb-4705-82f3-2e48e50cabf2","historyId":"a5b41ca06b9fbb486b9dd24106498508","fullName":"com.source.classes.invoices.poinvoice.checklist.PoInvChecklistRejectTest.reject","labels":[{"name":"package","value":"com.source.classes.invoices.poinvoice.checklist.PoInvChecklistRejectTest"},{"name":"testClass","value":"com.source.classes.invoices.poinvoice.checklist.PoInvChecklistRejectTest"},{"name":"testMethod","value":"reject"},{"name":"parentSuite","value":"GePS-HOPES_YEA PS Catalog Test Cases"},{"name":"suite","value":"PO Invoice Checklist Reject"},{"name":"subSuite","value":"com.source.classes.invoices.poinvoice.checklist.PoInvChecklistRejectTest"},{"name":"host","value":"Issac"},{"name":"thread","value":"19856@Issac.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"feature","value":"Purchase Order Invoice CheckList Reject"},{"name":"epic","value":"Purchase Order Invoice"},{"name":"severity","value":"normal"}],"links":[],"name":"Purchase Order Invoice CheckList Reject Test","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Invoice Checklist Reject was not Successful expected [0] but found [200]","trace":"java.lang.AssertionError: Invoice Checklist Reject was not Successful expected [0] but found [200]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1578)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:150)\r\n\tat org.testng.Assert.assertEquals(Assert.java:132)\r\n\tat org.testng.Assert.assertEquals(Assert.java:1419)\r\n\tat org.testng.Assert.assertEquals(Assert.java:1383)\r\n\tat com.source.classes.invoices.poinvoice.checklist.PoInvChecklistRejectTest.reject(PoInvChecklistRejectTest.java:18)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\r\n\tat org.testng.TestRunner.run(TestRunner.java:614)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1269)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","description":"Test Description: Verify Verifier Can Able To Reject The Purchase Order Invoice Checklist","steps":[],"attachments":[],"parameters":[{"name":"arg2","value":"33968e43-588d-46c4-a259-7d2f359762e5"},{"name":"arg1","value":"240025TRN03538"},{"name":"arg0","value":"240025INV00513"}],"start":1747911225010,"stop":1747911322965}